Sources/Sentry/include/SentryExtraPackages.h

```diff
@@ -0,0 +1,23 @@
+#if __has_include(<Sentry/SentryDefines.h>)
+#    import <Sentry/SentryDefines.h>
+#else
+#    import "SentryDefines.h"
+#endif
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface SentryExtraPackages : NSObject
+SENTRY_NO_INIT
+
++ (void)addPackageName:(NSString *)name version:(NSString *)version;
++ (NSMutableSet<NSDictionary<NSString *, NSString *> *> *)getPackages;
```

Comment:
`l`: I guess this can be a set.
```suggestion
+ (NSSet<NSDictionary<NSString *, NSString *> *> *)getPackages;
```

Reply:
I've done this on purpose to avoid one extra copy. We create a copy when getting the packages and then we need to add the SDK Package (https://github.com/getsentry/sentry-cocoa/pull/4637/files#diff-def4a3cae1e4126909fdbd84870ad3fb7b9922755d490980cc1e6f1a756a5ad9R43) so by making this mutable we avoid making another copy.